srcpkgs/speakersafetyd/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'speakersafetyd'
+pkgname=speakersafetyd
+version=1.0.2
+revision=1
+build_style=cargo
+hostmakedepends="alsa-lib pkg-config cargo"
+makedepends="alsa-lib-devel"
+short_desc="Software Smart Amp implementation"
+maintainer="Orphaned <orphan@voidlinux.org>"
+license="MIT"
+homepage="https://github.com/AsahiLinux/speakersafetyd"
+distfiles="https://github.com/AsahiLinux/speakersafetyd/archive/refs/tags/${version}.tar.gz"
+checksum=844ae3719c029e826f58c3799b6e358d189b0c42ade7a91f6c35b960cae35919
+make_dirs="/var/lib/speakersafetyd/blackbox 0755 root root"
+
+post_install() {
+	# can use make install-data if don't mind about systemd
+	vinstall 95-speakersafetyd.rules 644 usr/lib/udev/rules.d/
+
+	for f in conf/apple/*; do
+		vinstall "$f" 644 usr/share/speakersafetyd/apple
+	done
```

Comment:
personally, I disagree with not using `make install-data` here.

Doing this manually is pointless.

Installing the systemd service is completely harmless. Plenty of void packages ship with systemd services, and (afaik) removing from templates is discouraged unless there's a good reason to do so

Reply:
Sure, no big deal for me. I can change it.